<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbCoucou">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Coucou</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFermer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Fermer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCoucou">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>????</string>
    </property>
    <property name="buddy">
     <cstring>lblCoucou</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRecopier">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Recopier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>X ?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuMon_Menu">
    <property name="title">
     <string>Mon Menu</string>
    </property>
    <addaction name="actionCoucou"/>
    <addaction name="actionRecopier"/>
    <addaction name="actionFermer"/>
   </widget>
   <addaction name="menuMon_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionCoucou">
   <property name="text">
    <string>Coucou</string>
   </property>
  </action>
  <action name="actionRecopier">
   <property name="text">
    <string>Recopier</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
